content/en/blog/_posts/2023-10-18-gateway-api-time-to-switch-i2gw/index.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: blog
+title: "Time to Upgrade; Migrating to Gateway API"
+date: 2023-10-24T10:00:00-08:00
+slug: introducing-ingress2gateway
+---
+
+***Authors:*** Lior Lieberman (Google), Kobi Levi (independent)
+
+Today we are releasing [ingress2gateway](https://github.com/kubernetes-sigs/ingress2gateway), a tool
+that can help you migrate from [Ingress](/docs/concepts/services-networking/ingress/) to [Gateway
+API](https://gateway-api.sigs.k8s.io). Gateway API is just weeks away from graduating to GA, if you
+haven't upgraded yet, now's the time to think about it!
+
+
+## Background
+
+In the ever-evolving world of Kubernetes, networking plays a pivotal role. As more applications are
+deployed in Kubernetes clusters, effective exposure of these services to clients becomes a critical
+concern. If you've been working with Kubernetes, you're likely familiar with the [Ingress API],
+which has been the go-to solution for managing external access to services.
+
+[Ingress API]:/docs/concepts/services-networking/ingress/
+
+The Ingress API provides a way to route external traffic to your applications within the cluster,
+making it an indispensable tool for many Kubernetes users. Ingress has its limitations however, and
+as applications become more complex and the demands on your Kubernetes clusters increase, these
+limitations can become bottlenecks.
+
+Some of the limitations are:
+
+- **Insufficient common denominator** - by attempting to establish a common denominator for various
+  HTTP proxies, Ingress can only accommodate basic HTTP routing, forcing more features of
+  contemporary proxies like traffic splitting and header matching into provider-specific,
+  non-transferable annotations.
+- **Inadequate permission model** - Ingress spec configures both infrastructure and application
+  configuration in one object. With Ingress, the cluster operator and application developer operate
+  on the same Ingress object without being aware of each other’s roles. This creates an insufficient
+  role-based access control and has high potential for setup errors. 
+- **Lack of protocol diversity** - Ingress primarily focuses on HTTP(S) routing and does not provide
+  native support for other protocols, such as TCP, UDP and gRPC. This limitation makes it less
+  suitable for handling non-HTTP workloads.
+
+## Gateway API
+
+To overcome this, Gateway API is designed to provide a more flexible, extensible, and powerful way
+to manage traffic to your services.
+
+Gateway API is just weeks away from a GA (General Availability) release. It provides a standard
```

Comment:
Weeks? Or days?

Reply:
Target is Oct 31, could slip to Nov 1. I think that's on the low end for "weeks" and high end for "days". Open to what you prefer here.